apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -677,19 +721,31 @@ module.exports = {
           multiple: true,
         },
         {
-          name: "link",
-          label: "Link",
-          widget: "object",
+          name: "links",
+          label: "Links",
+          widget: "list",
           fields: [
             {
-              label: "Title",
-              name: "title",
-              widget: "string",
-            },
-            {
-              label: "Href",
-              name: "href",
-              widget: "string",
+              label: "Link",
+              name: "link",
+              widget: "object",
+              fields: [
+                {
+                  label: "slug",
+                  name: "slug",
+                  widget: "string",
```

Comment:
Does any item need a manually entered `slug` any more?

Reply:
For these two I think they have to be set but I'll confirm

![Screenshot 2022-07-21 at 11 53 32](https://user-images.githubusercontent.com/13068580/180173156-1db36e1c-106e-423b-868d-2ab423c4f317.png)

Reply:
[here](https://github.com/CodeForAfrica/ui/blob/main/apps/codeforafrica/src/components/ProjectDetails/ProjectDescriptionButton.js#L14) @kilemensi

Reply:
Cool. Looking at that code, the purpose of `slug` was to pick appropriate icon since icons are/were hard-coded. Now that we have CMS, shouldn't the icon be set here as part of the link?